### PR TITLE
[feature][doc] Add a note for restarting brokers with the load shedding strategy dynamically updated

### PR DESCRIPTION
**Pull request by momo-jun**
### Motivation

Add docs for #13074 

### Modifications

1. Add a note for restarting brokers with the load shedding strategy dynamically updated
2. Fix deprecated links for pulsar admin CLI and typos.

The preview looks good.

### Documentation
  
- [x] `doc` 
(Your PR contains doc changes)

### CODE REVIEW

**Comment by momo-jun** [comment]
Hi @rdhabalia @Jason918 @eolivelli  @Demogorgon314 can you pls review this doc PR? Thanks.
//cc @Anonymitaet

File: site2/docs/administration-load-balance.md [hunk 1]
```diff
@@ -143,6 +143,9 @@ loadBalancerSheddingGracePeriodMinutes=30
 
 Pulsar supports the following types of shedding strategies. From Pulsar 2.10, the **default** shedding strategy is `ThresholdShedder`.
 
+> **Note**<br />
```
Comment (Anonymitaet) [inline site2/docs/administration-load-balance.md:146]: do you want this layout? 
image

if so, you do not need to add <br />

if no and you want there is an empty line before the next paragrahy, you can try <br /><br />

Comment (momo-jun) [reply site2/docs/administration-load-balance.md:146]: Yes, I want this layout. But it only works with "br" added this time. Without it, it looks like below:
image

Not sure what's going wrong. I thought something had been changed in the framework.

Comment (Anonymitaet) [reply site2/docs/administration-load-balance.md:146]: may try another layout like:

```
>**Note**
>
>xxx
```

Comment (momo-jun) [reply site2/docs/administration-load-balance.md:146]: This solution leaves an empty line in between, which is not necessary. 
image

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**